### PR TITLE
Feature: update global nginx proxy container

### PR DESCRIPTION
**Pull request by defunctl**
https://hub.docker.com/r/jwilder/nginx-proxy is no longer maintained and is taken over by https://hub.docker.com/r/nginxproxy/nginx-proxy.

This upgrades the `tribe-proxy` image to the `1.0-alpine` branch.

### CODE REVIEW

File: config/app.php [hunk 1]
```diff
@@ -27,7 +27,7 @@
     */
 
     //'version' => app('git.version'),
-    'version' => '5.6.3',
+    'version' => '5.6.5',
```
Comment (tarecord) [inline config/app.php:30]: Skipping 5.6.4?

**Review by tarecord — APPROVED** [review]
Seems simple enough 🚀

File: storage/global/docker-compose.yml [hunk 2]
```diff
@@ -7,8 +7,8 @@ services:
     cap_add:
       - NET_ADMIN
     ports:
-      - '53:53/udp'
-      - '53:53/tcp'
+      - "53:53/udp"
```
Comment (ChrisMKindred) [inline storage/global/docker-compose.yml:10]: I like that these are consistent now with the `"` for all of them.

**Review by ChrisMKindred — APPROVED** [review]
👍

**Pull request merged**